etna/analysis/forecast/plots.py

```diff
@@ -95,7 +117,7 @@ def plot_forecast(
     _, ax = _prepare_axes(num_plots=len(segments), columns_num=columns_num, figsize=figsize)
 
     if prediction_intervals:
-        quantiles = _select_quantiles(forecast_results, quantiles)
+        prediction_intervals_names = _select_prediction_intervals_names(forecast_results, quantiles)
```

Comment:
What about error which can be raised here? We could add it into `Raises` block of docstring.

Reply:
Added to docstring